docker-compose.yml

```diff
@@ -104,35 +114,33 @@ services:
       - bootstrap.memory_lock=true
       # set to reasonable values on production
       - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    volumes:
+      - esdata:/usr/share/elasticsearch/data/elasticsearch
     ports:
-      - "9200:9200"
-      - "9300:9300"
+      - "9200"
+      - "9300"
 
   rabbitmq:
     restart: "always"
     image: rabbitmq
+    depends_on:
+     - redis
     ports:
       - "4369"
       - "5672"
 
   nginx:
     restart: "always"
     build: ./nginx
+    image: cod3/nginx
     ports:
       - "80:80"
+    environment:
+      - DEBUG=False
     volumes:
-      - "./cernopendata:/code/cernopendata"
-      - "./scripts:/code/scripts"
-    depends_on:
-     - static
-    volumes_from:
-      - static
-    links:
-      - web
+      - webdata:/usr/local/var/cernopendata/var/cernopendata-instance/static
 
-  static:
-    restart: "no"
-    build: .
-    volumes:
-      - /usr/local/var/cernopendata/var/cernopendata-instance/static
-    user: invenio
+volumes:
+  webdata:
```

Comment:
To keep naming harmony with containers: 

- webdata 
- postgresqldata
- elasticsearchdata